blockchains/cardano/cardano_worker.js

```diff
@@ -12,6 +12,7 @@ const DEFAULT_TIMEOUT_MSEC = parseInt(process.env.DEFAULT_TIMEOUT || '30000');
 class CardanoWorker extends BaseWorker {
   constructor(settings) {
     super(settings);
+    this.pRetry = null;
```

Comment:
Why not just leave it at undefined?
```js
this.pRetry;
```

Reply:
I thought this gives some clarity that such member variable would be later used, but obviously it is questionable. Removed.